src/tasks/Tasks.ts

```diff
@@ -639,13 +639,16 @@ export abstract class ATouringTask implements ITouringTask {
   }
 
   setLineupHighlight(cellData: IScoreCell, enable: boolean, cssClass: string) {
+
+    const focusedLineupNode = this.node.closest('.lu-taggle');//select the closest lineup node to highlight -`lu-taggle` is set in ARankingView
```

Comment:
I suggest to cache the d3 object here, since you are calling it 3x later on:

```suggestion
    const focusedLineupNode = d3.select(this.node.closest('.lu-taggle')); // select the closest lineup node to highlight -`lu-taggle` is set in ARankingView
```